requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 
 # For running doctests (during "python setup.py test")
 Sphinx >= 1.4.5
+sphinx_bootstrap_theme
```

Comment:
Yes, _sphinx_bootstrap_theme_ is required (the tests failed for me without it). As to _sphinx_rtd_theme_, I uninstalled it, ran `make clean html doctest` and all went fine. Are you sure it is required? Notice that it is commented out:

```
$ grep sphinx_rtd_theme docs/*
docs/conf.py:# html_theme = 'sphinx_rtd_theme'
```